core/state.go

```diff
@@ -60,6 +62,38 @@ type OperatorState struct {
 	BlockNumber uint
 }
 
+func (s *OperatorState) Hash() ([16]byte, error) {
+	marshalable := struct {
+		Operators   map[QuorumID]map[string]OperatorInfo
```

Comment:
Hashing on hashmap may be nondeterministic

Reply:
good point, I converted it to sorted slice